### PR TITLE
Bug 1141294 - Move BrowserWindow and sub UI classes into their own sub folder

### PR DESCRIPTION
**Pull request by gbrmachado**

### CODE REVIEW

File: firefox_puppeteer/ui/browser/browserwindow.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (whimboo) [inline firefox_puppeteer/ui/browser/browserwindow.py]: Please rename this file just to `windows.py`. The `browser` part is already part of the folder name.

File: firefox_puppeteer/ui/browser/browserwindow.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                let chromeWindow = arguments[0].ownerDocument.defaultView;
+                return PrivateBrowsingUtils.isWindowPrivate(chromeWindow);
+            """, script_args=[self.window_element])
+
+    @property
+    def navbar(self):
+        """Provides access to the navigation bar. This is the toolbar containing
+        the back, forward and home buttons. It also contains the location bar.
+
+        See the :class:`~ui.toolbars.NavBar` reference.
```
Comment (whimboo) [inline firefox_puppeteer/ui/browser/browserwindow.py]: You will have to get those references updated too.

File: firefox_puppeteer/ui/browser/browserwindow.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+            navbar = self.window_element.find_element(By.ID, 'nav-bar')
+            self._navbar = NavBar(lambda: self.marionette, self, navbar)
+
+        return self._navbar
+
+    @property
+    def tabbar(self):
+        """Provides access to the tab bar.
+
+        See the :class:`~ui.tabbar.TabBar` reference.
```
Comment (whimboo) [inline firefox_puppeteer/ui/browser/browserwindow.py]: Same here for the reference update.

**Comment by whimboo** [comment]
This update looks great! Nicely done Goma! I only see some smaller issues which would need an update here. See my individual comments.

File: firefox_puppeteer/docs/index.rst [hunk 4]
```diff
@@ -36,8 +36,9 @@ future. Each library is available from an instance of the FirefoxTestCase class.
    ui/about_window/window
    ui/menu
    ui/pageinfo/window
-   ui/tabbar
-   ui/toolbars
+   ui/browser/tabbar
+   ui/browser/toolbars
+   ui/browser/windows
```
Comment (whimboo) [inline firefox_puppeteer/docs/index.rst]: Oh, sorry! Looks like somehow an additional 's' made it into the suggestion of my last review comments. This should have been `window.py` similar to the page info window as seen right above. Sorry for that.

**Comment by gbrmachado** [comment]
Ok whimboo Np. :) I've changed the names

**Comment by gbrmachado** [comment]
I'm getting this error:
https://travis-ci.org/mozilla/firefox-ui-tests/jobs/83381219#L1052-L1064

I think it's not related with my modifications

**Comment by whimboo** [comment]
Yes, that failure is not related to your PR and is also only happening intermittently. I will get your PR merged to mozilla-central in a moment.

**Comment by whimboo** [comment]
Merged PR to master as: https://github.com/mozilla/firefox-ui-tests/commit/99715a8fcdfc109d6d992f47e9d80298947b2076

**Pull request closed without merging**